mobile/app/(app)/(drawer)/(tabs)/(observation)/form-details/[formId].tsx

```diff
@@ -138,14 +138,23 @@ const FormDetails = () => {
           answer = { ...answer, selectionValues: selectedAnswersTexts };
         }
 
+        // sort the notes by createdAt date in order to extract the text of the last added note
+        const sortedNotes =
+          notes && notes[q.id]
+            ? notes[q.id].slice().sort((a, b) => {
+                // added unary '+' operator to avoid ts error that the right-hand side of an arithmetic operation must be of type 'any', 'number', 'bigint' or an enum type
```

Comment:
It is not about the TS error but being able to substract 2 numbers as an arithmetic operation. Using the "+" unary operator will try to convert the Date to a Number by calling `valueOf` which is the same as `.getTime()`